packages/cli/src/cmds/new.ts

```diff
@@ -84,7 +84,9 @@ export default class New extends YargsCommand {
     const core = result.value;
 
     {
-      const result = await core.createProject(getSystemInputs());
+      const inputs = getSystemInputs();
+      inputs.projectId = uuid.v4();
```

Comment:
I thought it should core to generate the projectId. Should not be generated by client. And no need to put "projectId" property in Inputs.

Reply:
ok